edxval/api.py

```diff
@@ -765,3 +819,24 @@ def import_from_xml(xml, edx_video_id, course_id=None):
             'bitrate': encoded_video_el.get('bitrate'),
         })
     create_video(data)
+    create_transcript_objects(xml)
+
+
+def create_transcript_objects(xml):
+    """
+    Create VideoTranscript objects.
+
+    Arguments:
+        xml (Element): lxml Element object
+    """
+    for transcript in xml.findall('.//transcripts/transcript'):
+        try:
+            VideoTranscript.create_or_update(
```

Comment:
I think we decided to not to overwrite/update existing video transcripts?

Reply:
Discussed with @muzaffaryousaf and decided that we will update existing transcripts upon import.

Do you see any issues?

Reply:
users will be able to manipulate existing transcripts by editing the xml.